apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Tests/Logging/GoogleLoggerTest.cs

```diff
@@ -67,10 +67,12 @@ private string Formatter(string message, Exception ex)
             Common.LoggerOptions options = Common.LoggerOptions.CreateWithServiceContext(
                 logLevel, logName, labels, monitoredResource, retryOptions: retryOptions, serviceName: serviceName, version: version);
 #pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Do we still need this pragma?

Reply:
Yes, this is the one that is used to keep backward compatibility.